__init__.py

```diff
@@ -1,2 +1,4 @@
 __VERSION__ = "v0.0.1"
```

Comment:
`v1.0.0` ?

Reply:
ah, there's another one with correct value in `theiavalidate/__init__.py`, is `__init__.py` still needed? its commit message says it's a hack to make import work 2 years ago

Reply:
I'm honestly not sure if it's still needed lol -- I'll update this one just in case. thanks for pointing this out!